packages/firebase/firestore/memory/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '@firbease/firestore/memory';
```

Comment:
This file provides the entry point to memory only Firestore build. We then have a build target that takes this file as the input. `packages/firebase/firestore/memory/package.json` will point its main field to the output of the build,  so developers can import memory only firestore like following
```
import 'firebase/firestore/memory';
```

Reply:
This is not a recommend way to import Firebase SDKs though, is it? I thought we pushed users to use `@firebase/firestore/memory`.

Also - while we are at it, how does this effect the end user experience/build?

Reply:
We have been telling people to install `firebase` package, instead of individual `@firebase` packages to avoid incompatible version. As a result, we use `firebase/xxx` in our guides as well.

It should not have any impact on end user experience/build. no change required on devevlopers end.